regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -22,9 +22,7 @@ var CommentReviewView = Backbone.View.extend({
     this.docId = this.$el.data('doc-id');
     this.template = _.template($('#comment-template').html());
 
-    this.mode = 'preview';
-    this.previewUrl = null;
-
+    this.previewLoading = false;
```

Comment:
Don't think so--requesting a preview disables the preview button, so users shouldn't get racing preview requests unless they start messing with the console. Passing a loading flag to render could also lead to bugs--for example, if we allow users to delete comments from this view, deleting a comment would trigger a re-render, and if a preview was loading, we'd need to make sure to pass the flag to render. Unless we store loading state somewhere (javascript state, dom, looking at pending ajax requests), we're not going to know whether to render the preview button in loading or not-loading state at the point, unless I'm misunderstanding your suggestion.